site/content/contribute/more-info/mobile/build-your-own/_index.md

```diff
@@ -16,7 +16,9 @@ As an alternative we've also created a set of **scripts** to help automate build
 
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
-{{<note "Note:">}}
+{{</note>}}
```

Comment:
@varghesejose2020 - Please delete the note syntax on line 19. The note title is set via line 17.